sdk/storage/azure-storage-quickquery/src/main/java/com/azure/storage/quickquery/models/BlobQuickQueryError.java

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.quickquery.models;
+
+/**
+ * Defines a blob quick query error that can be returned on parsing a blob quick query request.
+ */
+public class BlobQuickQueryError {
+
+    private boolean fatal;
+    private String name;
+    private String description;
+    private long position;
+
+    /**
+     * Creates a new BlobQuickQueryError object.
+     * @param fatal Whether or not the error is fatal.
+     * @param name The name of the error.
+     * @param description A description of the error.
+     * @param position The blob offset at which the error occurred.
+     */
+    public BlobQuickQueryError(boolean fatal, String name, String description, long position) {
+        this.fatal = fatal;
```

Comment:
I wonder what's example of "fatal" that we don't want to just throw exception. If it's fatal then don't we want to stop the operation?